tasks.robot

```diff
@@ -66,22 +66,26 @@ GitHub MFA
 
 
 Google MFA
-    # Open browser as normal usage.
-    @{args} =    Create List
-    ...    --user-agent=Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/92.0.4515.107 Safari/537.36
-    ...    --disable-dev-shm-usage    --no-sandbox
-    &{options} =    Create Dictionary    arguments    ${args}
-    Open Available Browser    https://accounts.google.com/ServiceLogin
-    ...    options=${options}
-    # Doesn't work even if you attach to your existing browser and profile.
-    # Attach Chrome Browser    9222
-    # Go To    https://accounts.google.com/ServiceLogin
-
-    # This gets to a page detecting and blocking automation.
-    Input Text When Element Is Visible    id:identifierId    ${SECRETS}[google_usr]
-    Click Element When Visible    //span[contains(text(), 'Next')]
+    # Login as close as you can to usual human browsing. (by attaching to a normally
+    #  opened browser)
+    Evaluate    setattr(selenium.webdriver, "Chrome", undetected_chromedriver.Chrome)
+    ...    modules=selenium.webdriver,undetected_chromedriver
+    Open Browser    https://accounts.google.com/ServiceLogin    browser=Chrome
```

Comment:
Used this instead of `Open Available Browser` because the internally set `"excludeSwitches"` experimental option will break the `undetected_chromedriver.Chrome` instantiation. (and I feel we won't patch our library just to be able to disable it, for now at least)